js/chips.js

```diff
@@ -16,18 +17,47 @@ function selectChip(chip) {
 	selectedChip = chip; // set the newly-clicked chip to be the global selectedChip
 
 	return selectedChip; // return the freshly-set selectedChip variable back to the global scope
-	
+
 	// - get the ID of the current square that the selected chip is in
 	// - check the surrounding squares for available valid moves
 	// - call the validateMoves function, pass the selected chip and the current square as arguments.
 	//   might also need to pass-in the player or player's chip color
 	// validateMoves(selectedChip, currentSquare);
 }
 
+function selectSquare(square) {
```

Comment:
This function is essentially a copy of the function `selectChip()`. It will set the global `var selectedSquare` if a chip has already been selected. Right now this function includes some console.logs for my testing purposes.